src/EA.Weee.RequestHandlers.Tests.Unit/Admin/AatfReports/GetAllAatfObligatedDataCsvHandlerTests.cs

```diff
@@ -142,6 +145,7 @@ public async Task GetAllAatfObligatedDataCsvHandler_StringParameters_ReturnsFile
 
             // Assert
             Assert.Contains("2019_A1_B2C", data.FileName);
+            Assert.Contains(datetime, data.FileName);
```

Comment:
Change the assert to not contains but the actual file name as we should know what it is... Can use the SystemTIme freeze